types/node/process.d.ts

```diff
@@ -1,7 +1,9 @@
 declare module "process" {
     import { Control, MessageOptions } from "node:child_process";
     import * as tty from "node:tty";
+    import * as net from "node:net";
     import { Worker } from "node:worker_threads";
+    type SendHandle = net.Socket | net.Server | undefined;
```

Comment:
The documentation is poor as ever. [Here](https://github.com/nodejs/node/blob/5dd478c338dbc2f9ee1e185c25ff75ac8d169c5a/lib/internal/child_process.js#L780-L792) is what's considered valid input.